packages/client/Atmosphere.ts

```diff
@@ -304,6 +304,22 @@ export default class Atmosphere extends Environment {
       if (!data) console.error('QueryMap is incomplete. Run `yarn clean`')
     }
     const transport = uploadables ? this.baseHTTPTransport : this.transport
+    const errorCheckerSink: Sink<any> | undefined = sink
+      ? {
+          next: (value) => {
+            if (value.errors) {
+              console.error(value.errors)
+            }
+            return sink.next(value)
+          },
+          error: (error: Error, isUncaughtThrownError?: boolean) =>
```

Comment:
+1 nit, took me awhile to see that the only thing changed is the next handler. could we do something like 

```ts
const errorSink = sink ? {...sink, next: (val) => {}} : undefined
```